components/classroom/main.js

```diff
@@ -42,3 +45,50 @@ window.onload = () => {
         console.log('Project ID not found in URL');
     }
 };
+
+const ROLE_HIERARCHY = [Roles.OWNER, Roles.LEADER, Roles.CONTRIBUTOR];
+
+// identifies the highest privilege role from a list of user roles and returns the top-ranked valid role
+function getHighestPrivilegeRole(userRoles) {
```

Comment:
Maybe in future sprints we have userRoles as an attribute of a User object, just so we start to see some connection between all of these structures!

Reply:
Yes we will be working on them shortly and to confirm I just want to clarify the role attached to a user it may be any of the 3 roles we have